include/fields2cover/route_planning/route_planner_base.h

```diff
@@ -56,7 +56,7 @@ class RoutePlannerBase {
   ///          between two nodes.
   /// @param d_tol Tolerance distance to consider if two points are the same.
   virtual F2CGraph2D createCoverageGraph(
-      const F2CCells& cells, const F2CSwathsByCells& swaths_by_cells,
+      const F2CCells& cells, const F2CSwathsByCells& swaths_by_cells, bool redirect_swaths,
```

Comment:
This change will prevent compilation of previous code. Add the bool parameter at the end and add a default value that maintains the same behavior as now.